modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java

```diff
@@ -150,18 +142,12 @@ protected String getDDMFormFieldValue(
 			ddmFormFieldTypeServicesRegistry.getDDMFormFieldValueRenderer(
 				ddmFormField.getType());
 
-		Stream<DDMFormFieldValue> stream = ddmFormFieldValues.stream();
-
 		return _html.unescape(
 			StringUtil.merge(
-				stream.map(
+				TransformUtil.transform(
+					ddmFormFieldValues,
 					ddmForFieldValue -> ddmFormFieldValueRenderer.render(
-						ddmForFieldValue, locale)
-				).filter(
-					Validator::isNotNull
```

Comment:
@JanisZhang Here `Validator.isNotNull` is applied on strings, so it will check whether the string is `null`, an empty string, a whitespace, or `"null"`.

So we need to keep this usage.